src/Test/UnitTests/attributes.jl

```diff
@@ -31,6 +31,26 @@ function silent(model::MOI.ModelLike, config::TestConfig)
 end
 unittests["silent"] = silent
 
+"""
+    time_limit_sec(model::MOI.ModelLike, config::TestConfig)
+
+Test that the [`MOI.TimeLimitSec`](@ref) attribute is implemented for `model`.
+"""
+function time_limit_sec(model::MOI.ModelLike, config::TestConfig)
+    if config.solve
+        @test MOI.supports(model, MOI.TimeLimitSec())
+        # Get the current value to restore it at the end of the test
+        value = MOI.get(model, MOI.TimeLimitSec())
```

Comment:
This `get()` throws an error if `TimeLimitSec` is not `set` to `nothing` before. It's done in `test/Test/unit.jl` for now.